iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/tests/iothub/twin/UpdateTwinTests.java

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Copyright (c) Microsoft. All rights reserved.
+ *  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+ */
+
+package com.microsoft.azure.sdk.iot.common.tests.iothub.twin;
+
+import com.microsoft.azure.sdk.iot.common.helpers.ClientType;
+import com.microsoft.azure.sdk.iot.common.helpers.ConditionalIgnoreRule;
+import com.microsoft.azure.sdk.iot.common.helpers.StandardTierOnlyRule;
+import com.microsoft.azure.sdk.iot.common.setup.iothub.DeviceTwinCommon;
+import com.microsoft.azure.sdk.iot.device.IotHubClientProtocol;
+import com.microsoft.azure.sdk.iot.device.exceptions.ModuleClientException;
+import com.microsoft.azure.sdk.iot.service.auth.AuthenticationType;
+import com.microsoft.azure.sdk.iot.service.devicetwin.*;
+import com.microsoft.azure.sdk.iot.service.exceptions.IotHubException;
+import com.microsoft.azure.sdk.iot.service.exceptions.IotHubPreconditionFailedException;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.security.GeneralSecurityException;
+import java.util.*;
+
+/**
+ * Test class containing all tests to be run on JVM and android pertaining to Queries. Class needs to be extended
+ * in order to run these tests as that extended class handles setting connection strings and certificate generation
+ */
+public class UpdateTwinTests extends DeviceTwinCommon
```

Comment:
Extending DeviceTwinCommon will make your tests in this file run with parameterized inputs on device client protocols, which doesn't seem relevant to a service client only test. Could you remove this extension and add in the parts that you need from it manually instead?

Reply:
sure